manifests/server.pp

```diff
@@ -40,6 +40,7 @@
   $topology                 = 'subnet',
   $custom_options           = [],
   $ccd                      = 'ccd',
+  $compress                 = 'legacy',
```

Comment:
Is there a more modern compression method to prefer here as a default?

Reply:
Good question, I don't know, I did not want to change the defaults.
Look at https://openvpn.net/community-resources/reference-manual-for-openvpn-2-4/ (search for compress) there is mentioned that comp-lzo is deprecated, but still valid.
We could probably set the hiera option on a per distribution basis, or create a fact that holds the openvpn version.
I read somewhere (but cannot the find the link right now, that compression is not secure as uncompressed ... (I decided to not use any compression).

Reply:
Hmm, well what is here doesn't change the behavior for the default case.